src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -141,7 +141,6 @@ get_os_info(boost::property_tree::ptree &pt)
   pt.put_child("libraries", _ptLibInfo);
 
   // The file is a requirement as per latest Linux standards
-  // https://www.freedesktop.org/software/systemd/man/os-release.html
```

Comment:
How will that help?

Reply:
I am just curious about the level of obfuscation is required. :-)
Is it a scan of the comments or a scan of the executable code?
Otherwise
```
See software/systemd/man/os-release.html at freedesktop dot org
```
At the end I do not understand the rationale for the rebuttal since there is no code coming from there. Just some documentation to look at to understand better the code.

Reply:
All source code is scanned, it is not just comments :-)

Reply:
What about
```
// See man os-release
```
or something like that?